dev/reference/transactions/transaction-model.md

```diff
@@ -68,4 +68,8 @@ Due to its distributed nature, workloads that are single-threaded or latency-sen
 
 + Transaction
 
-    When TiDB is in the execution of loading data, by default, a record with 20,000 rows of data is seen as a transaction for persistent storage. If a load data operation inserts more than 20,000 rows, it will be divided into multiple transactions to commit. If an error occurs in one transaction, this transaction in process will not be committed. However, transactions before that are committed successfully. In this case, a part of the load data operation is successfully inserted, and the rest of the data insertion fails. But MySQL treats a load data operation as a transaction, one error leads to the failure of the entire load data operation.
+    When TiDB executes the `LOAD DATA` operation, a record with 20,000 rows of data is seen as a transaction for persistent storage by default. If one `LOAD DATA` operation inserts more than 20,000 rows, it is divided into multiple transactions to commit. If an error occurs in one transaction, this transaction in process is not committed. However, transactions before that are committed successfully. In this case, a part of the `LOAD DATA` operation is successfully inserted, and the rest of the data insertion fails. But MySQL treats a `LOAD DATA` operation as one transaction, one error leads to the failure of the entire `LOAD DATA` operation.
+
+    > **Warning:**
+    >
+    > The `LOAD DATA` operation in TiDB does not guarantee the atomicity of transactions, so it is not recommended to use it in production environment.
```

Comment:
You can use it in production for loading data. You can't regularly use it once data is loaded.
```suggestion
    > The `LOAD DATA` operation in TiDB does not guarantee the atomicity of transactions, so it recommended only for initial batch loading data, and not during regular usage for production environment(s).
```